### PR TITLE
Add board

### PR DESCRIPTION
**Pull request by lrharris215**
The game now prints an empty board to the console when run. It will then print another board with some discs already placed on it, to show off the different colors.

### CODE REVIEW

File: src/main/java/com/connectfour/Display.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        System.out.println(message);
+    }
+    // for chars
+    public void print(char message){
+        System.out.print(message);
+    }
+    public void println(char message){
+        System.out.println(message);
+    }
+
```
Comment (kudukacar) [inline src/main/java/com/connectfour/Display.java:26]: I didn't see any unit tests for Display. In any case, how will you test Display if you don't use dependency injection?  See this example:  https://github.com/kudukacar/server/blob/master/src/main/java/infrastructure/Logger.java and this https://github.com/kudukacar/server/blob/master/src/test/java/infrastructure/LoggerTest.java

File: src/test/java/com/connectfour/DiscTest.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    void testGetSymbol() {
+        Disc testDisc = new Disc("blue", 'X');
+        assertEquals('X', testDisc.getSymbol());
+    }
+
+    @Test
+    void testGetColor() {
+        Disc testDisc = new Disc("blue", 'X');
+        assertEquals("blue", testDisc.getColor());
+    }
```
Comment (kudukacar) [inline src/test/java/com/connectfour/DiscTest.java]: I don't think you need to test Disc.  It's just a data object.  It does not have any methods (besides getters).

File: src/test/java/com/connectfour/NullDiscTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        NullDisc testNull = NullDisc.getNullDisc();
+        assertEquals('\u25CC', testNull.getSymbol());
+    }
+
+    @Test
+    void getColor() {
+        NullDisc testNull = NullDisc.getNullDisc();
+        assertEquals("null", testNull.getColor());
+    }
+}
```
Comment (kudukacar) [inline src/test/java/com/connectfour/NullDiscTest.java]: Same comment as above regarding tests for Disc.

**Comment by kudukacar** [comment]
You're already a Java expert.  Lots of good stuff here.  Nice use of interfaces!

File: src/main/java/com/connectfour/Display.java (hunk 1 above)
Comment (lrharris215) [reply src/main/java/com/connectfour/Display.java:26]: Good point, originally had it like that, then changed it. Will change it back

**Review by kudukacar — APPROVED** [review]
Looks Good!

**Pull request merged**